src/components/thumbnail-maker/TagItem.tsx

```
@@ -1,22 +1,26 @@
+interface Props {
   tag: Tag;
   onRemove: () => void;
   onClick: () => void;
   className?: string;
-}) {
+}
+
+export function TagItem({ tag, onRemove, onClick, className }: Props) {
+  const paletteStyle = useCurrentPaletteStyle();
   return (
     <div className={cn("group relative", className)} onClick={onClick}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)